### PR TITLE
When using content from SSG, use the DataStream by default

### LINKED ISSUE #41: Use SSG datastreams instead of SSG XCCDFs
When user customizes SSG profiles in SCAP Workbench, he customizes datastreams. When he saves tailoring file, he the file references the datastream. That's a complication when he will want to use it in OSCAP Anacoda Addon via kickstart, because OSCAP Anaconda Addon uses XCCDFs instead, so the  the original profile will not be found, and empty profile will be applied as a result.

### PR DESCRIPTION
**Pull request by yuumasato**
When loading content from SCAP Security Guide, we should use content from the DataStream, not from XCCDF file.

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1465402
Also fixes #41

### CODE REVIEW

File: org_fedora_oscap/common.py [hunk 1]
```diff
@@ -52,12 +52,12 @@
 TARGET_CONTENT_DIR = "/root/openscap_data/"
 
 SSG_DIR = "/usr/share/xml/scap/ssg/content/"
-SSG_XCCDF = "ssg-rhel7-xccdf.xml"
+SSG_SDS = "ssg-rhel7-ds.xml"
```
Comment (redhatrises) [inline org_fedora_oscap/common.py]: Please make this SSG_CONTENT

**Comment by redhatrises** [comment]
@yuumasato Tests are still broken because of this change.

File: org_fedora_oscap/common.py (hunk 1 above)
Comment (yuumasato) [reply org_fedora_oscap/common.py]: Oops.. will change

**Comment by yuumasato** [comment]
Yeah, I'll fix the tests and try to add some more.

**Comment by dahaic** [comment]
Thank you @yuumasato . I have just tried the installation with updated OAA, and this change successfully fixed the issue reported by BZ.

**Pull request merged**